src/components/shared/TreeView.js

```diff
@@ -170,6 +173,8 @@ type TreeViewRowScrolledColumnsProps<
   +onClick: (NodeIndex, SyntheticMouseEvent<>) => mixed,
   +onAppendageButtonClick?: ((NodeIndex | null, string) => mixed) | null,
   +highlightRegExp: RegExp | null,
+  +rowHeightStyle: { height: CssPixels, lineHeight: string },
```

Comment:
Maybe write here in a comment (and/or below line 388 where we define the object) that React will automatically use the `px` unit for `height` but not for `lineHeight`. (I had to look it up)